frontend/components/qrCode.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useQRCode } from 'next-qrcode';
+
+export default function qr(gameId: string) {
```

Comment:
I think that the GitHub check is failing on this... should you capitalize it to Qr? Doing that would also follow what is outlined in convention.md.

Reply:
The frontend test will fail no matter what due to the use of next.router on the index page, which is the only test currently in main.  But I will go ahead and capitalize Qr

Reply:
oops I misunderstood, it is fixed now thanks